qiskit/visualization/utils.py

```diff
@@ -208,5 +175,150 @@ def _get_gate_span(qregs, instruction):
 
     if instruction.cargs:
         return qregs[min_index:]
+    if instruction.condition and isinstance(instruction.condition[0], ClassicalRegister):
```

Comment:
Can `instruction.condition` ever be anything other than a ClassicalRegister?

Reply:
I don't know. I've seen simulations in which there are tests on qubits. It's not possible at the current state of the art; it may not be possible in quantum mechanics! But I wasn't sure.

If you're saying a condition in Qiskit will _never_ be other than a classical register, I'll remove the test.

Reply:
We are looking at a DAGNode here, and per the [docstring](https://github.com/Qiskit/qiskit-terra/blob/35abeb9da0f8f975931a8f8bd281a64d6df0ba82/qiskit/dagcircuit/dagnode.py#L78) I don't think it can be anything other than a ClassicalRegister

Reply:
Done.